pkg/eks/api.go

```diff
@@ -187,6 +187,9 @@ func newAWSProvider(spec *api.ProviderConfig, configurationLoader AWSConfigurati
 	}
 
 	if spec.Region == "" {
+		if cfg.Region == "" {
+			return nil, fmt.Errorf("no region code: please set the AWS region to ~/.aws/config file or environment variables")
```

Comment:
also I'd prefer this error message: `"AWS Region must be set, please set the AWS Region in AWS config file or as AWS_REGION environment variable"`

Reply:
I'll change this message : ) Thank you